R/messages.R

```diff
@@ -4,58 +4,176 @@
 # Tableau.
 warning_message <- function() {
   message_contents <- NULL
-  # If running on RSC, perform checks; if not return NULL
-  if (check_rstudio_connect()) {
-    # RStudio Connect Details
-    # Provide messaging if RSC:
-    #  * Is a version that doesn't support Tableau Extensions
-    #  * Isn't configured to support Tableau Extensions
-    #  * Doesn't have Server.Address configured
-
-    # Connect to RStudio Connect API and read server settings
-    rsc_client <- connect()
-    settings <- rsc_client$server_settings()
-
-    # Server.Address
-    connect_server <- Sys.getenv("CONNECT_SERVER")
-
-    # Does this installation support Tableau Extensions
-    connect_support <- settings$tableau_integration_enabled
-
-    # Find connect version
-    connect_version <- settings$version
-
-    if (is.null(connect_support)) {
-      message_contents <- paste(message_contents,
-                                paste0("> **WARNING**: This version of RStudio Connect (",
-                                       connect_version,
-                                       ") does not support Tableau Analytics Extension APIs. Please upgrade RStudio Connect.\n"),
-                                sep = "\n")
-    } else if (!connect_support) {
-      message_contents <- paste(message_contents,
-                                "> **WARNING**: Tableau Analytics Extension API support is currently disabled in RStudio Connect's configuration.\n",
-                                sep = "\n")
-    }
+  message_count <- 0
+
+  server <- Sys.getenv("CONNECT_SERVER", NA_character_)
+  "!DEBUG Environment Variable CONNECT_SERVER = `server`"
+
+  if (is.na(server) || server == "") {
+    "!DEBUG Problem: CONNECT_SERVER not defined within environment variables"
+
+    message_contents <- paste0(
+      message_contents,
+      paste0("### The environment variable *CONNECT_SERVER* is not defined.",
```

Comment:
This part isn't properly rendering with OpenAPI at the moment

Reply:
Changes pushed up!

![2021-07-29 at 11 10 AM](https://user-images.githubusercontent.com/17675905/127543729-cd60e4cc-5db1-47b3-897d-efacecb4db7a.jpg)